contract_isp/wizard/activate_contract_service.py

```diff
@@ -1,51 +1,38 @@
 # -*- coding: utf-8 -*-
-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    Copyright (C) 2013 Savoirfaire-Linux Inc. (<www.savoirfairelinux.com>).
-#    Copyright (C) 2011-Today Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>)
-
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2013 Savoirfaire-Linux Inc. (<www.savoirfairelinux.com>).
+# © 2015-Today Serpent Consulting Services Pvt. Ltd.
+#    (<http://www.serpentcs.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
 import datetime
-from openerp import models, fields, api
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from openerp import api, fields, models, _
+
+
+@api.one
+def _get_account_id(self):
+    if self._context.get('active_model', '') == 'contract.service':
+        contract_id = self._context.get('active_id')
+        contract_service = self.env['contract.service'].browse(contract_id)
+        return contract_service.account_id.id
+    return None
+
+
+@api.one
```

Comment:
@JayVora-SerpentCS Better to avoid using `@api.one` since it's deprecated in newer versions of odoo